app/config/message.js

```diff
@@ -51,8 +53,10 @@ const mqConfig = {
     topic: process.env.RETURN_TOPIC_ADDRESS,
     type: 'subscription'
   },
-  statementTopic: {
-    address: process.env.STATEMENT_TOPIC_ADDRESS
+  statementSubscription: {
+    address: process.env.STATEMENT_SUBSCRIPTION_ADDRESS,
+    topic: process.env.STATEMENT_TOPIC_ADDRESS,
```

Comment:
Will need new env vars for the new topic and subscription.

Top tip, can't have spaces before `_TOPIC_ADDRESS` and `_SUBSCRIPTION_ADDRESS`

So can have `DATA_TOPIC_ADDRESS` but not `STATEMENT_DATA_TOPIC_ADDRESS for Jenkins dynamic reasons reasons.